moveit_ros/moveit_servo/src/servo_node.cpp

```diff
@@ -110,23 +110,25 @@ ServoNode::ServoNode(const rclcpp::NodeOptions& options)
   pose_subscriber_ = node_->create_subscription<geometry_msgs::msg::PoseStamped>(
       servo_params_.pose_command_in_topic, 10, std::bind(&ServoNode::poseCallback, this, std::placeholders::_1));
 
-  // Create publisher for joint trajectory message
-  trajectory_publisher_ = node_->create_publisher<trajectory_msgs::msg::JointTrajectory>(
-      servo_params_.command_out_topic, rclcpp::SystemDefaultsQoS());
+  if (servo_params_.command_out_type == "trajectory_msgs/JointTrajectory")
+    trajectory_publisher_ = node_->create_publisher<trajectory_msgs::msg::JointTrajectory>(
+        servo_params_.command_out_topic, rclcpp::SystemDefaultsQoS());
+  else if (servo_params_.command_out_type == "std_msgs/Float64MultiArray")
+    multi_array_publisher_ = node_->create_publisher<std_msgs::msg::Float64MultiArray>(servo_params_.command_out_topic,
```

Comment:
We should have an "else" that logs an error with invalid type -- unless this is validated elsewhere?

Reply:
It is validated by generate_parameter_library